examples/SuchThatTest.php

```diff
@@ -37,6 +37,24 @@ function($n) {
             ->then($this->allNumbersAreBiggerThan(42));
     }
 
+    public function testSuchThatAcceptsPHPUnitConstraints()
+    {
+        $this->forAll(
```

Comment:
@giorgiosironi 

I'm aware it's the convention in this project, but how about wrapping like this instead:

```php
$this
    ->forAll(Generator\vector(
        5,
        Generator\filter(
            function($n) {
                return $n > 42;
            },
            Generator\choose(0, 1000)
        )
    ))
    ->then($this->allNumbersAreBiggerThan(42))
;
```